src/engine/BMGame.php

```diff
@@ -4084,7 +3851,7 @@ protected function setBMPlayerProps($property, array $value, $subclass = '') {
     /**
      * Set all players' waitingOnAction statuses to FALSE
      */
-    protected function setAllToNotWaiting() {
+    public function setAllToNotWaiting() {
```

Comment:
Arguably this is a symptom that there's way too much game logic in BMInterface.  My first cut at this would be to say that anywhere BMInterface is directly calling `$game->log_action()`, that's a symptom that a BMGame function is needed to handle that case.  (At a glance, that seems right --- BMGame calls log_action() 12 times, while BMInterface calls it 5 times.  That matches my instinct that the 12 times are correct and the 5 times are wrong.)

On the one hand, that's orthogonal to this pull, so you should create a new issue and not handle this here.  On the other hand, you're making a function public which shouldn't be public, in this pull, because of that issue.  So i'm torn about whether it should be fixed now.